content/24.vaccines-novel.md

```diff
@@ -184,12 +175,11 @@ The concern with these early development projects was immunogenicity, not safety
 Around the turn of the millennium, a hepatitis B vaccine development program demonstrated that these vaccines can induce both antibody and cellular immune response [@doi:10/c9ntfp].
 As of 2018, however, only two plasmid-vectored DNA vaccines had been approved for commercial use, and both were for veterinary populations [@doi:10.5772/intechopen.76754].
 
-**Applications to COVID-19:**
-Several plasmid-vectored DNA vaccines have been developed against COVID-19 (Table @tbl:approved-DNA).
-In fact, the ZyCoV-D vaccines developed by India's Zydus Cadila is the first plasmid-vectored DNA to receive approval [@doi:10.1016/j.vaccine.2020.10.056; @doi:10.1038/d41586-021-02385-x].
-<!--It would be great to go into ZyCoV-D since it made it to approval, but we originally put our effort into covering INO-4800. Consider updating post-submission?-->
+##### Applications to COVID-19
 
-Similarly, another plasmid-vectored DNA vaccine, INO-4800 [@clinicaltrials:NCT04336410], was developed by Inovio Pharmaceuticals Technology that uses electroporation as an adjuvant.
+Several plasmid-vectored DNA vaccines have been developed against COVID-19 (Table @tbl:approved-DNA).
+In fact, the ZyCoV-D vaccines developed by India's Zydus Cadila is the first plasmid-vectored DNA vaccine to receive approval or to be used in human medicine [@doi:10.1016/j.vaccine.2020.10.056; @doi:10.1038/d41586-021-02385-x; 10.1001/jama.2021.16625].
```

Comment:
[This](https://jamanetwork.com/journals/jama/fullarticle/2784978) specifically says it is the "first clinical use" of a "DNA vaccine" in humans, but that can't be right, because viral-vectored vaccines were used before 8/2021

Reply:
I think that's correct, they are saying it is because it is the first use of a plasmid of bacterial origin that has been approved in humans (plenty preclinical stuff out there) - at least up to 2018 none were approved in humans 10.3390/medsci6020027

Reply:
Thank you Ronan!!!